packages/playground-server/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: "3"
+services:
+  postgres:
+    image: postgres:11.2-alpine
+    ports:
+      - "5432:5432"
```

Comment:
This won't be persistent, will it? Don't we need to mount a volume?

Reply:
https://github.com/counterfactual/monorepo/pull/1171